corehq/apps/users/templates/users/partials/edit_role_modal.html

```diff
@@ -487,6 +487,44 @@ <h4 class="modal-title" data-bind="text: modalTitle"></h4>
                 </div>
               </div>
             </div>
+            <div class="form-group">
```

Comment:
profiles is a domain privilege so I think this permission should be displayed only if the domain has the privilege. it's something like 
`domain_has_privilege(self.domain, privileges.APP_USER_PROFILES)`

Reply:
made this change using https://github.com/dimagi/commcare-hq/blob/fb0a93e0988adf2f0d3bd5d68e732b9f965c9e12/corehq/apps/hqwebapp/templatetags/hq_shared_tags.py#L744